pkg/cpu/cpu.go

```diff
@@ -17,11 +17,40 @@
 package cpu
 
 import (
-	"fmt"
 	"sync"
 	"time"
 )
 
+// rollingAvg holds the rolling average of the cpu load on the minio
+// server over its lifetime
+var rollingAvg *Load
+
+// triggers the average load computation at server spawn
+func init() {
+	rollingAvg = &Load{
+		Min: float64(0),
+		Max: float64(0),
+		Avg: float64(0),
+	}
+	var rollingSum float64
+	var cycles float64
+	go func() {
+		for {
+			<-time.After(5 * time.Second)
```

Comment:
`time.Sleep` is more readable since `time.After()` and `time.Sleep()` are underneath the same thing and also move the `5 * time.Second` as a global constant with an appropriate name.

Reply:
Done